### PR TITLE
v0/getmempoolinfo added

### PR DESCRIPTION
**Pull request by aitoribanez**

### CODE REVIEW

**Comment by FrancisPouliot** [comment]
Thanks for the PR

File: proxy_docker/app/script/requesthandler.sh [hunk 1]
```diff
@@ -284,6 +284,13 @@ main()
           response_to_client "${response}" ${?}
           break
           ;;
+        getmempoolinfo)
+          # curl GET http://192.168.111.152:8080/getmempoolinfo
+          trace "trace: $(get_mempool_info)"
+          response=$(get_mempool_info "${line}")
```
Comment (Kexkey) [inline proxy_docker/app/script/requesthandler.sh]: get_mempool_info is called twice.  Please trace response variable or data variable inside the get_mempool_info function.

Comment (Kexkey) [inline proxy_docker/app/script/requesthandler.sh]: You don't need to pass $line to get_mempool_info

**Comment by Kexkey** [comment]
Hi @aitoribanez !  Thanks for the changes, it's good now.

Before merging, can you add the new endpoint to the openapi doc please?

https://github.com/SatoshiPortal/cyphernode/blob/dev/doc/openapi/v0/cyphernode-api.yaml

You can take the get_blockchain_info as an example, it's that simple.

Thanks!

**Pull request merged**